src/ocm/components/clusterConfiguration/networkConfiguration/NetworkConfiguration.tsx

```diff
@@ -215,12 +218,30 @@ const NetworkConfiguration = ({
     [cluster.cpuArchitecture, cluster.openshiftVersion, featureSupportLevelData, isDualStack],
   );
 
+  const dispatch = useDispatch();
+  const toggleManagedNetworking = React.useCallback(
+    async (checked: boolean) => {
+      if (checked) {
+        if (cluster.id) {
```

Comment:
How come the cluster does not have an id at this point? :)~

Reply:
True. At this point the cluster id can't be null